dev/storage.h

```diff
@@ -1014,7 +1125,8 @@ namespace sqlite_orm {
                 std::map<std::string, sync_schema_result> result;
                 auto db = con.get();
                 this->impl.for_each([&result, db, preserve, this](auto& tableImpl) {
-                    auto schemaStatus = this->schema_status(tableImpl, db, preserve);
+                    bool attempt_to_preserve = true;
+                    auto schemaStatus = this->schema_status(tableImpl, db, preserve, attempt_to_preserve);
```

Comment:
`attempt_to_preserve` is passed here but never used. Better accept is as a pointer not ref to `schema_status` and pass `nullptr` when we don't need it

Reply:
ok